collections/ansible_collections/massopencloud/esi/roles/floating_ip/tasks/main.yaml

```diff
@@ -0,0 +1,17 @@
+- name: Validate floating_ip_state parameter
```

Comment:
validation can be done through argument specs https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_reuse_roles.html#role-argument-validation

Reply:
Oh, neat; I've updated the role to use argument validation.